src/pages/studies/SidebarFilters.tsx

```diff
@@ -1,21 +1,23 @@
 /* eslint-disable react/display-name */
 import React from 'react';
-import { InfoCircleOutlined, SearchOutlined } from '@ant-design/icons';
+import { InfoCircleOutlined, ReadOutlined } from '@ant-design/icons';
 import FilterContainer from '@ferlab/ui/core/components/filters/FilterContainer';
 import { IFilter } from '@ferlab/ui/core/components/filters/types';
-import { Input, Tooltip } from 'antd';
+import { ISqonGroupFilter } from '@ferlab/ui/core/components/QueryBuilder/types';
+import { Col, Row, Tooltip } from 'antd';
 
 import history from 'services/history';
-import { SidebarData } from 'store/graphql/studies/actions';
+import { SidebarData, useGetStudiesSearch } from 'store/graphql/studies/actions';
 
-import { getFilterType, getSelectedFilters, updateFilters, updateQueryFilters } from './utils';
+import SearchBar from './SearchBar';
+import { getFilterType, getSelectedFilters, updateFilters } from './utils';
 
 import style from './SidebarFilter.module.scss';
 
-const keyEnhance = (key: string) => {
+const keyEnhance = (key: string, s: string = ' No Data') => {
   switch (key) {
     case '__missing__':
```

Comment:
not your code but: I think __missing__ is put in a constant in the portal or in arranger (that you can import)